var/spack/repos/builtin/packages/py-slepc4py/package.py

```diff
@@ -11,13 +11,15 @@ class PySlepc4py(PythonPackage):
     """
 
     homepage = "https://gitlab.com/slepc/slepc4py"
-    url      = "https://slepc.upv.es/download/distrib/slepc4py-3.16.2.tar.gz"
+    url      = "https://slepc.upv.es/download/distrib/slepc4py-3.17.1.tar.gz"
```

Comment:
BTW: this update is not needed [as the URL is just the template . It just adds to code churn - for each such update]

Reply:
Ok